apps/test/unit/javalab/PlaygroundTest.js

```diff
@@ -394,14 +447,31 @@ describe('Playground', () => {
   function createSampleImageDetails(filename, id) {
     return {
       filename: filename,
-      x: 0,
-      y: 0,
-      width: 100,
-      height: 50,
+      x: '0',
```

Comment:
All of the values stored in Redux / provided from Javabuilder are strings, so I thought I'd update our test data to reflect that.